packages/react-native/local-cli/runMacOS/runMacOS.js

```diff
@@ -19,10 +36,10 @@ const {
 
```

Comment:
We also depend on `@react-native-community/cli-tools` without declaring it in `package.json`. Do we want to add it as an explicit dependency or resolve it via `@react-native-community/cli`?

cc @acoates-ms

Reply:
https://github.com/microsoft/react-native-macos/pull/1993